README.md

```diff
@@ -43,6 +43,12 @@ const options = {
    * defaults to false
    */
   booleansAsIntegers: false,
+
+  /**
+   * don't add brackets [] to File list
```

Comment:
There will be a configuration option for dot notation instead of bracket notation. Can we name this flag something else to make it sound more compatible?

Reply:
@therealparmesh I'm not sure which new configuration you are talking about...

This option is only about a list of `File` types. How about `noFileCollectionBrackets`?